OpenXmlPowerTools/OpenXMLWordprocessingMLToHtmlConverter/WmlToHtmlConverter.cs

```diff
@@ -279,6 +279,143 @@ private static void SetStyleElementValue(XElement xhtml, string styleValue)
             }
         }
 
+        private static object? ListAwareConvertToHtmlTransform(WordprocessingDocument wordDoc,
```

Comment:
It general a good idea to order methods in a feed style. ListAwareConvertToHtmlTransform is placed in line 282, but it should be placed after the last usage, in that case after line 2866